### ISSUE TITLE
Fix FP S6754 (`hook-use-state`): Ignore state variables without a setter

### ISSUE THREAD

**Issue by yassin-kammoun-sonarsource** [SonarSource/SonarJS#4627]
The rule reports an issue when the destructured return value of `useState` only includes the state variable without the corresponding setter. This seems like an antipattern but it's outside the scope of the rule since it should only care about symmetrical naming.

```javascript
const [value /*, missing setValue */] = useState(...);
```
